cdap-distributions/src/emr/install.sh

```diff
@@ -22,6 +22,8 @@
 # The git branch to clone
 CDAP_BRANCH=${CDAP_BRANCH:-release/4.1}
 # Optional tag to checkout - All released versions of this script should set this
+# like this: CDAP_TAG=${CDAP_TAG:+tag} as this allows setting tag to empty/null
+# otherwise, it should be CDAP_TAG=''
 CDAP_TAG=''
```

Comment:
I've gone ahead and made this change to match HDInsight. Keeping them the same makes sense.